pkg/runtime/runtime.go

```diff
@@ -1632,10 +1639,6 @@ func (a *DaprRuntime) initActors() error {
 	return err
 }
 
-func (a *DaprRuntime) hostingActors() bool {
```

Comment:
We can't remove this function for 1.5.1 since it's being used by `runtime.go` in that branch...